.github/workflows/goja-node.js.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will do a clean install of node dependencies, build the source code,
+# and run linting/tests across different versions of node.
+# For more information see:
+#   https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Goja Node.js CI
+
+defaults:
+  run:
+    working-directory: ldk/goja
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 10.x, 12.x, 14.x ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm set audit false
+      - run: npm ci
+      - run: npm audit --production --audit-level=low
+      - run: npm run build --if-present
+      - run: npm run lint
+      - run: npm run test
+      # - run: npm run doc
```

Comment:
TODO: Add documentation?

Reply:
I've been thinking about this; I think we will probably want to generate documentation again with Typedoc to the LDK/go folder in the GH pages repo. There's aspects of the current Typedoc presentation I don't love but we don't need to solve that right now.

Reply:
I will create a card for this.

Reply:
https://crosschx.atlassian.net/browse/SIDE-2043